Makefile

```diff
@@ -69,6 +69,11 @@ universe-test: dagger-debug # Run universe tests
 	yarn --cwd "./pkg/universe.dagger.io" install
 	TESTDIR=$(UNIVERSE_TESTDIR) DAGGER_BINARY="$(shell pwd)/cmd/dagger/dagger-debug" yarn --cwd "./pkg/universe.dagger.io" test
 
+.PHONY: experimental-universe-tests
+experimental-universe-tests: dagger-debug # Run experimental universe tests
+	yarn --cwd "./pkg/universe.dagger.io" install
+	TESTDIR=$(UNIVERSE_EXPERIMENTAL_TESTDIR) DAGGER_BINARY="$(shell pwd)/cmd/dagger/dagger-debug" yarn --cwd "./pkg/universe.dagger.io" test-experimental
```

Comment:
Unless I'm missing something, `UNIVERSE_EXPERIMENTAL_TESTDIR` doesn't exist. If we make the change mentioned in my other comment, it shouldn't need to either so I think we can just remove this.

Reply:
Agree. I removed it.